viewer/whylogs-script.js

```diff
@@ -0,0 +1,547 @@
+"use strict";
+
+(function () {
+  var MESSAGES = {
+    error: {
+      noInputElementFound: "It seems we could not find the file input element.",
+      noBrowserSupport: "This browser does not seem to support the `files` property of file inputs.",
+      noFileSelected: "Please select a file.",
+      fileAPINotSupported: "The file API is not supported on this browser yet.",
+      invalidJSONFile:
+        "It seems the JSON file you are trying to load is not valid due because it does not match whylogs format. Please check the file validity and try again.",
```

Comment:
```suggestion
        "The JSON file you are trying to load is does not match the expected whylogs format. Please check the file and try again.",
```

Reply:
There's an extra `is` here :)

`The JSON file you are trying to load does not match the expected whylogs format. Please check the file and try again`